source/lex.tex

```diff
@@ -474,12 +475,13 @@
 The appearance of either of the characters \tcode{'} or \tcode{\textbackslash} or of
 either of the character sequences \tcode{/*} or \tcode{//} in a
 \grammarterm{q-char-sequence} or an \grammarterm{h-char-sequence}
-is conditionally-supported with implementation-defined semantics,
-as is the appearance of the
-character \tcode{"} in an \grammarterm{h-char-sequence}.\footnote{Thus, a
-sequence of characters that resembles an escape sequence might result in an
-error, be interpreted as the character corresponding to the escape sequence, or
-have a completely different meaning, depending on the implementation.}%
+is conditionally-supported with \impldef{meaning of \tcode{'}, \tcode{\textbackslash},
+\tcode{/*}, or \tcode{//} in a \grammarterm{q-char-sequence} or an
+\grammarterm{h-char-sequence}} semantics, as is the appearance of the character
+\tcode{"} in an \grammarterm{h-char-sequence}.\footnote{Thus, a sequence of characters
```

Comment:
We should presumably have some entry for the `"`-in-*h-char-sequence* case, perhaps add words for this case to the prior \impldef?